### ISSUE TITLE
HALT TempCompare measurement validation errors

### ISSUE THREAD

**Issue by jweber-ice** [WIPACrepo/stf#62]
Review relative temperature measurement failures.

**Comment by jweber-ice** [comment]
Files on cobalt:

```
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151624/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 5.4069999999999965

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_151702/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 6.169999999999995

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155602/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 5.632000000000005

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 8.908900000000003

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_155627/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.5319999999999965

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162547/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 5.087400000000002

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_162552/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.631900000000002

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 8.257899999999992
--
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_170412/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.0589999999999975

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_173421/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 6.950900000000004

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180832/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 5.596999999999994

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_180835/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 8.167299999999997

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffMag": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffMag",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_183805/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": "nan"

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191806/degg-0c52f6f87fbd/TempCompare-base-v1.0_degg-0c52f6f87fbd.json-                    "measured_value": 5.125

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_191830/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 9.751800000000003

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_195713/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.178899999999999

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 7.626400000000004
--
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffPres": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffPres",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_203114/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.945000000000007

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_214226/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 9.156899999999993

/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                "TempDiffAcc": {
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "name": "TempDiffAcc",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json:                    "outcome": "FAIL",
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "validators": [
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                        "-5.0 <= x <= 5.0"
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    ],
/data/exp/IceCubeUpgrade/commissioning/stf/halt/HALT/2020.02.14_221550/degg-2f8c8eef6792/TempCompare-base-v1.0_degg-2f8c8eef6792.json-                    "measured_value": 5.310899999999997
```

**Comment by jweber-ice** [comment]
Closed by commit 6ca24c77ba5ca6378462cdc91f79fc3badc91b71
